lib/moon_web/pages/design/popover_page.ex

```diff
@@ -0,0 +1,267 @@
+defmodule MoonWeb.Pages.Design.PopoverPage do
+  @moduledoc false
+
+  use MoonWeb, :live_view
+
+  alias Moon.Design.Popover
+  alias Moon.Design.Button
+  alias Moon.Design.Button.IconButton
+
+  alias MoonWeb.Components.Page
+  alias MoonWeb.Components.ComponentPageDescription
+  alias MoonWeb.Components.ExampleAndCode
+  alias MoonWeb.Components.PropsTable
+
+  alias MoonWeb.Pages.Design.Popover.Content.Default
```

Comment:
please replace this aliases with only one - alias MoonWeb.Pages.Design.Popover.Content and than use them as Content.Default, Content.Position, ... in the code
also these imports should be shown in examples, bc noone knows what are these components